unit_tests/fvm/test_gradient.cpp

```diff
@@ -1,19 +1,129 @@
 #include "aw_test_utils/MeshTest.H"
 #include "amr-wind/fvm/gradient.H"
+#include "aw_test_utils/iter_tools.H"
 
 namespace amr_wind_tests {
 
 class FvmOpTest : public MeshTest
 {};
 
+
+namespace {
+
+amrex::Real phi_eval(amrex::Real x,amrex::Real y, amrex::Real z, int index)
+{
+    amrex::Real phi[3];
+
+    phi[0] = 5.0*x*y*z + 4.0*x*x + 3.0*y*y - 2.3*z*z + 1.3*y*z + 3.8*x*z + 9.4*x*y + 3.4*x+ 2.0*x + 3.14;
```

Comment:
Would it make sense to have coefficients generated by random number generator than hard code here?